endpoint.py

```diff
@@ -0,0 +1,88 @@
+from fastapi import FastAPI, Depends, HTTPException, Query
+from sqlmodel import Session, select
+from sqlmodel_init.database_and_session import *
+from typing import List
+# sqlmodel_databaseからインポートしたモデルからテーブルを生成
+
+
+app = FastAPI()
+
+# sqlomodelを作成？
+@app.on_event("startup")
+def on_startup():
+    create_db_and_tables()
+
+# Taskモデルにレコードが存在するかチェックしてなければ指定したstatus_codeでエラーをraiseする関数
+def get_task_or_exception(task_id: int, status_code, session: Session):
+    task_db_record = session.get(Task, task_id)
```

Comment:
```suggestion
    task_by_id = session.get(Task, task_id)
```
些細な命名規則かもしれませんが、`task_id`を使用して`Task`を取得しているので
変数名を上記のように編集してtaskをどのように取得したかを書くのもありかもしれません！
`Session`から取得するのはDBからのレコード以外考えずらいかもな、、、と思った次第です！

Reply:
ありがとうございます。
この変数名は一括で変えさせていただきました。